docs/pages/pmd/userdocs/tools/tools.md

```diff
@@ -197,7 +197,7 @@ To install the PMD plugin for Eclipse:
 *   Start Eclipse and open a project
 *   Select "Help"->"Software Updates"->"Find and Install"
 *   Click "Next", then click "New remote site"
-*   Enter "PMD" into the Name field and <https://dl.bintray.com/pmd/pmd-eclipse-plugin/updates/> into the URL field
+*   Enter "PMD" into the Name field and select the URL of the version you need from [here](https://pmd.github.io/pmd-eclipse-plugin-p2-site/) and into the URL field
```

Comment:
Note, you don't need to use a specific version URL, just use "https://pmd.github.io/pmd-eclipse-plugin-p2-site/" ... it will find all the versions that are available.

```suggestion
*   Enter "PMD" into the Name field and <https://pmd.github.io/pmd-eclipse-plugin-p2-site/> into the URL field
```